tests/test_schedules.py

```diff
@@ -2603,6 +2603,20 @@ def axpy(n: size, x: f32[n], y: f32[n]):
     assert str(axpy) == golden
 
 
+def test_stage_mem_out_of_bound_point(golden):
+    @proc
+    def foo(n: size, m: size, x: f32[n], y: f32[n]):
+        assert m >= n
+        for i in seq(0, m):
+            if i < n:
+                y[i] = x[i]
+            if i < n:
```

Comment:
Is there reason we need two `if i < n: y[i] = x[i]` stmts for this test?

Reply:
That's a good point. I can't think of why I wrote it this way lol.

Reply:
Just fixed it.